parlai/scripts/train_model.py

```diff
@@ -543,7 +546,7 @@ def validate(self):
             )
             self.best_valid = new_valid
             self.impatience = 0
-            if opt.get('model_file') and is_primary_worker():
+            if opt.get('model_file'):
```

Comment:
just making sure I understand - we can get rid of this check because it's handled in `save_model` right?

Reply:
We need to be able do save_on_nonprimary_worker actually